etc/trunion-dev.ini

```diff
@@ -80,3 +80,23 @@ certfile = trunion/tests/test_crt.jwk
 chainfile = trunion/tests/test_x509_chain.pem
 permitted_issuers = https://marketplace-dev.allizom.org
 we_are_signing = receipts
+
+[addons]
+ca_cert_file = /etc/trunion/addons_root_ca_cert.pem
```

Comment:
It's not actually used by ops for development or testing.  It's just an example to act as a guideline for ops.